libraries/consul_installation_binary.rb

```diff
@@ -32,7 +32,7 @@ def self.provides_auto?(_node, _resource)
       # @return [Hash]
       # @api private
       def self.default_inversion_options(node, resource)
-        extract_path = node.platform_family?('windows') ? node.config_prefix_path : '/opt/consul'
+        extract_path = windows? ? node.config_prefix_path : '/opt/consul'
```

Comment:
Seems like this hunk breaks the converge:
https://travis-ci.org/johnbellone/consul-cookbook/builds/254804735
```
undefined method `windows?' for ConsulCookbook::Provider::ConsulInstallationBinary:Class
```

Could you please try to replace the condition with `platform?('windows')` , as we do in other resources here?